contrib/lang/ess/packages.el

```diff
@@ -27,6 +27,16 @@ which require an initialization must be listed explicitly in the list.")
   ;; R --------------------------------------------------------------------------
   (eval-after-load "ess-site"
     '(progn
+       (add-to-list 'auto-mode-alist '("\\.R$" . R-mode))
+       (add-hook 'ess-mode-hook #'rainbow-delimiters-mode)
```

Comment:
Prefer to add `rainbow-delimiters` to `ess-packages` and define `ess/init-rainbow-delimiters` where you add the hook.
Example here with python and flycheck: https://github.com/syl20bnr/spacemacs/blob/c557111fd88342216202f29312fe7ad4996f9454/contrib/lang/python/packages.el#L133-134